css/store.css

```diff
@@ -0,0 +1,289 @@
+header {
+  height: 100px;
+  display: flex;
+  position: relative;
+}
+header :nth-child(2) {
+  margin-left: auto;
+  gap: 50px;
+  margin-right: 30px;
+}
+.burger-button {
+  display: none;
```

Comment:
Nice Question!
"The burger button is hidden because it's unnecessary when the page is displayed in the web view (for larger screens). I activate the burger button for mobile displays using media queries. This way, it's only visible and usable when the page is viewed on a mobile device."